### DISCUSSION TITLE
Add ability to allow editing of `belongs_to` association in the presence of `via_resource_class`.

### DISCUSSION THREAD

**Discussion by MrJoy** [avo-hq/avo#806] [Feature requests]
I'd like to make it easier to correct incorrect associations.  I mostly notice incorrect associations when I'm looking at a record and have a has_many list of its associated records.

The current workflow winds up being that I need to detach the record, then find it in the main index view for its resource type, edit it, and set the association properly.

I'd like to be able to just see the record in the has_many list, edit it, correct the association, save, then be returned to the show view that I was originally on.

I'm slightly worried that I've just played a really bad game of Who's On First with the above, so let me restate that using entities from my current project:

I clicked on a Toyline, and saw that one of the Items in my collection was associated with it, but I know that that Item is actually from a different Toyline and so I want to correct the error.  I click edit on the Item, and the Toyline dropdown is disabled.  So, I need to go to the Items link on the sidebar, find the Item in the list, and edit it from there.  Or I can detach it from the Toyline's show view, but then I still need to go to the Items link on the sidebar, find the Item in the list, and edit it.

**Comment by adrianthedev** [comment]
Fixed by https://github.com/avo-hq/avo/pull/811
